newdle/client/src/components/answer/AnswerPage.js

```diff
@@ -200,6 +212,12 @@ export default function AnswerPage() {
                 />
               </Segment>
             )}
+            <Header as="h4" className={styles.date}>
+              Newdle timezone: {newdleTz}
+            </Header>
+            <Header as="h4" className={styles.date}>
```

Comment:
Should we display this in case it's the same as `newdleTz`? And I'm not sure if `Header` (especially when rendered as a h4 or any other hX element) is semantically correct - it's not really a header..

Reply:
1) Yes, I believe we should still display both of them, to prevent confusion from people
2) I agree the semantical meaning is not correct, so I'll change it to something that semantically makes more sense

Reply:
Done, take a look at it now